aten/src/ATen/native/native_functions.yaml

```diff
@@ -6475,6 +6475,36 @@
     SparseCPU, SparseCUDA: norm_sparse
   autogen: native_norm.ScalarOpt_dim_dtype_out
 
+# TODO: add these to core aten opset
+# return values are: output, save_mean, save_var, reserve
```

Comment:
You can just put these names on the output. It will even automagically create a namedtuple in python that ill be returned from your function